presto-docs/src/main/sphinx/release/release-340.md

```diff
@@ -0,0 +1,64 @@
+# Release 340 (XX Aug 2020)
+
+## General Changes
+
+* Add support for query parameters in `LIMIT`, `OFFSET` and `FETCH FIRST` clauses. ({issue}`4529`, {issue}`4601`)
+* Add {func}`bitwise_left_shift`, {func}`bitwise_right_shift` and {func}`bitwise_right_shift_arithmetic`. ({issue}`772`)
+* Add `IF EXISTS `and `IF NOT EXISTS` syntax to `ALTER TABLE`. ({issue}`4651`)
+* Improve dynamic partition pruning for short queries. ({issue}`4388`}
+* Reduce cluster load by cancelling query stages from which data is no longer required. ({issue}`4290`)
+* Reduce query memory usage by improving retained size estimation for `VARCHAR` and `CHAR` types. ({issue}`4123`)
+* Improve query performance for queries containing ``starts_with`` predicates. ({issue}`4669`)
+* Improve performance of queries that use `DECIMAL` data type. ({issue}`4730`)
+* Fix failure when `GROUP BY` clause contains duplicate expressions. ({issue}`4609`)
+* Fix potential hang during query planning ({issue}`4635`).
+
+## Security
+
+* Verify access to referenced table in `CREATE TABLE LIKE`. ({issue}`4472`)
```

Comment:
Shouldn;'t this say what is being fixed? like "Fix unprivileged access to table's schema via CREATE TABLE LIKE"?